pkg/daemon/update.go

```diff
@@ -714,8 +715,18 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig, skipCertifi
 		return err
 	}
 
+	crioOverrideConfigmapExists := false
+	_, err = dn.kubeClient.CoreV1().ConfigMaps(ctrlcommon.MCONamespace).Get(context.TODO(), constants.ImageRegistryDrainOverrideConfigmap, metav1.GetOptions{})
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			klog.Errorf("Error fetching image registry drain override configmap: %v", err)
```

Comment:
question: If we encounter this error, is it something we should return early for? Or is just logging it sufficient?